dblp-aliases.csv

```diff
@@ -1363,6 +1363,7 @@ John Wylie Lloyd,John Lloyd
 John W. Lloyd,John Lloyd
 Jack M. Carroll,John M. Carroll
 John Philip Collomosse,John P. Collomosse
+John Dickerson,John P. Dickerson

```

Comment:
This one is spurious, though you need one for "John Dickerson 0001":
<img width="462" alt="Screen Shot 2021-03-17 at 6 14 58 PM" src="https://user-images.githubusercontent.com/1612723/111545624-b8a4bb80-874c-11eb-9768-73aa478ed7d8.png">

Reply:
Oh, good catch.  Thanks; it should be resolved now.